Automattic/wp-calypso
# Stats Widget: add widget entry in Odyssey Stats

Related to #75109

Works with https://github.com/Automattic/jetpack/pull/29772

## Proposed Changes

- Add `widget.jsx` to render the stats card of the WP.com home page as an example for the WP Admin widget.
- Added loader for widget to optimize loading time of WP-Admin Dashboard
- Add building entry for `widget-loader.js`.

## Testing Instructions

* Open your local Jetpack site.
* Check out the branch of the Jetpack PR: https://github.com/Automattic/jetpack/pull/29772.
* Build Jetpack if necessary: `jetpack build plugins/jetpack`.
* Run `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev` under `calypso/apps/odyssey-stats`.
* Navigate to `/wp-admin`.
* Ensure the bar chart is rendered in the Jetpack Stats widget.

<img width="1253" alt="image" src="https://user-images.githubusercontent.com/1425433/228455478-a9b6e225-9df6-4d3c-8939-11d1e53e05f9.png">

## Pre-merge Checklist
